packages/core/src/dialects/cockroachdb/query-generator.ts

```diff
@@ -0,0 +1,98 @@
+import type { TruncateOptions } from 'src/model';
+import { joinSQLFragments } from '../../utils/join-sql-fragments';
+import type { ListSchemasQueryOptions } from '../abstract/query-generator';
+import type { TableNameOrModel } from '../abstract/query-generator-typescript';
+import type { QueryWithBindParams } from '../abstract/query-generator.types';
+import { ENUM } from '../postgres/data-types';
+import { PostgresQueryGenerator } from '../postgres/query-generator';
+
+export class CockroachDbQueryGenerator extends PostgresQueryGenerator {
+
+  describeTableQuery(tableName: TableNameOrModel): string {
```

Comment:
The key differences are 

1. Replace `|| c.character_maximum_length` with `|| CAST(c.character_maximum_length AS STRING)` to avoid concatenation error between string and integer. The || is needed to avoid replacing in the wrong place.
2. Change unimplemented table `pg_statio_all_tables` to `pg_class`.
3. Change the unimplemented column `relid` to `oid`.
4. Aggregate enums in sort order by replacing `array_agg(e.enumlabel)` with `array_agg(e.enumlabel ORDER BY e.enumsortorder ASC)`